Don't use -Werror
That causes newer clang compilers to fail with:
error: lambda capture 'silent' is not used [-Werror,-Wunused-lambda-capture]
--- Makefile.orig	2017-09-22 15:01:50.000000000 -0500
+++ Makefile	2017-11-14 08:24:04.000000000 -0600
@@ -18,7 +18,7 @@
 DEFINES+=-DGIT_HASH=\"$(GIT_HASH)\"
 endif
 
-WARNFLAGS=-Werror -Wall -Wextra -Wunreachable-code
+WARNFLAGS=-Wall -Wextra -Wunreachable-code
 INCLUDES=-Iinclude -Istfl -Ifilter -I. -Irss
 BARE_CXXFLAGS=-std=c++11 -ggdb $(INCLUDES)
 CXXFLAGS+=$(BARE_CXXFLAGS) $(WARNFLAGS) $(DEFINES)
